reactive/kotlinx-coroutines-reactor/src/ReactorContext.kt

```diff
@@ -0,0 +1,11 @@
+package kotlinx.coroutines.reactor
+
+import reactor.util.context.Context
+import kotlin.coroutines.AbstractCoroutineContextElement
+import kotlin.coroutines.CoroutineContext
+
+class ReactorContext(val context: Context) : AbstractCoroutineContextElement(ReactorContext) {
```

Comment:
All public API should be properly documented, with corresponding examples.
A good example of such documentation is `ThreadLocal<T>.asContextElement` API.

I am very interested in examples of how this API is supposed to be used, maybe a @sdeleuze could point a few pseudocode-like examples?

Reply:
@SokolovaMaria note that both examples should be present: one with accessing reactor context from coroutine builder and the other with passing coroutine context from suspend function (or `Flow`?) to reactor entity